.ci/Jenkinsfile

```diff
@@ -344,15 +344,26 @@ def scpr(workspace, connection, remote_src, local_dst){
   }
 }
 
+def getPythonPath(version) {
+  sh(script: "curl https://pyenv.run | bash", label: "Install pyenv", returnStatus: true)
+  withEnv(["PATH+PYENV=${WORKSPACE}/.pyenv/bin"]){
+    sh(script: "pyenv install ${version}", label: "Install python with pyenv")
+    sh(script: "pyenv global ${version}", label: "Configure python with pyenv")
+    return sh(script: 'echo $(pyenv root)/shims', label: "Gather python path", returnStout: true)
+  }
+}
+
 /*
  * Runs python in venv
  */
 def pyrun(command){
   def awsProps = getVaultSecret(secret: "${AWS_PROVISIONER_SECRET}")
   def awsAuthObj = awsProps?.data
+  def pythonPath = getPythonPath('3.9.12')
```

Comment:
@v1v candidate to be added to the shared library as `withPython(version: '3.9.12') { ... }`?